examples/compute/region-disk.gyro

```diff
@@ -0,0 +1,50 @@
+google::compute-region-disk region-disk-example
+    name: "region-disk-example"
+    description: "region-disk-example-desc"
+    region: "us-central1"
+
+    replica-zones: [
+         "us-central1-c",
+         "us-central1-a"
+    ]
+
+    size-gb: 32
+    type: "pd-ssd"
+    labels: {
+        label-key: 'label-value'
+    }
+    physical-block-size-bytes: 16384
+
+    disk-encryption-key
+        raw-key: "disk-256-bit-raw-key"
```

Comment:
Give a valid raw key in the example so that the example runs out of the box. Do this for all examples using encryption keys across example and documentation.